src/materials/ShadowMaterial.js

```diff
@@ -35,6 +36,11 @@ function ShadowMaterial() {
 		}
 	} );
 
+	this.setValues( parameters );
```

Comment:
As a matter of practice, this is usually the last line of the constructor, so values the user sets are not overwritten.